src/Symfony/Component/HttpKernel/DataCollector/LoggerDataCollector.php

```diff
@@ -160,14 +161,25 @@ private function sanitizeLogs($logs)
             }
 
             $exception = $log['context']['exception'];
-            $errorId = md5("{$exception->getSeverity()}/{$exception->getLine()}/{$exception->getFile()}\0{$log['message']}", true);
+
+            if ($exception instanceof SilencedErrorContext) {
+                $log += array(
+                    'errorCount' => $exception->count,
+                    'scream' => true,
+                );
+                $sanitizedLogs[$log['message']] = $log;
+
+                continue;
```

Comment:
if we keep 1/100 as I just proposed, this will need to account for this possibility